utils/mkhtml.py

```diff
@@ -79,8 +79,8 @@ def decode(bytes_):
 src_file = "%s.html" % pgm
 tmp_file = "%s.tmp.html" % pgm
 
-trunk_url = "https://github.com/OSGeo/grass/tree/main/"
-addons_url = "https://github.com/OSGeo/grass-addons/tree/master/"
+trunk_url = "https://github.com/OSGeo/grass/tree/releasebranch_8_0/"
```

Comment:
Looks good but could the version be anyhow retrieved? (like further down in this file:)

https://github.com/OSGeo/grass/blob/7395fb869fff7350d306130df9ece345495af7ee/utils/mkhtml.py#L418

Reply:
Yes, you are right, Markus. thanks. I've already fixed it.